### ISSUE TITLE
icon, desktop file

### ISSUE THREAD

**Issue by alexmyczko** [Dakostu/MiniStocks#5]
Hi

A ministocks.desktop file with an icon would be great, so people can start it from a graphical menu, instead of from a terminal/shell, no?

BTW, it's in Debian and Ubuntu now, on all these architectures:
https://buildd.debian.org/status/package.php?p=ministocks

Should you want to have a link for binaries, this one is probably good:
https://repology.org/project/ministocks/versions

**Comment by Dakostu** [comment]
@alexmyczko 
Thank you again for your Debian support.

I whipped up a logo:

![Screenshot_20200209_183617](https://user-images.githubusercontent.com/29815238/74106948-a2045c00-4b6b-11ea-8ab5-5abf9bd08fa6.png)

Check out the branch [application-icons](https://github.com/Dakostu/MiniStocks/tree/application-icons) and see whether the changes are adequate to your liking.
It works on KDE at least:
![Screenshot_20200209_183228](https://user-images.githubusercontent.com/29815238/74106913-2d312200-4b6b-11ea-880c-29c96bb4e54e.png)

I also changed the target binary name from "MiniStocks" to "ministocks" in the project file and moved the supplied Makefile into the source code root folder. Does this cause any inconveniences on your end?
